src/main/java/net/sf/jabref/importer/fileformat/BibtexImporter.java

```diff
@@ -31,11 +31,11 @@
 public class BibtexImporter extends ImportFormat {
 
     /**
-     * @return false as that does not cause any harm in the current implementation of JabRef
+     * @return true as we have no effective way to decide whether a file is in bibtex format or not
      */
     @Override
     public boolean isRecognizedFormat(InputStream in) throws IOException {
-        return BibtexParser.isRecognizedFormat(new InputStreamReader(in));
+        return true;
```

Comment:
I do not like that we remove the logic to detect a bibtex file. On the contrary, we should extend this as we also want to detect whether a file is a bibtex or a biblatex file. This should not be removed.